src/vue/components/popup.vue

```diff
@@ -516,11 +516,13 @@ export default {
         .on('resizemove', this.resizeListener)
     }
 
-    const boundsCheck = this.isWithinBounds()
-    if (!boundsCheck.withinBounds) {
-      this.shift.x += boundsCheck.adjX
-      this.shift.y += boundsCheck.adjY
-      this.$options.logger.log(`Popup position has been adjusted to stay within the viewport`)
+    if (typeof this.$el.querySelector === 'function') {
```

Comment:
why this test here?

Reply:
We have similiar check before `interact.js` initialization inside panel.
So I added the same check here for consistency.